docs/protos.md

```diff
@@ -0,0 +1,33 @@
+# in-toto Statement protos
+
+Protobuf definitions for the in-toto statement type and some predicates are provided in the
+spec/ directory along with the pre-generated Go implementation of those protos.
```

Comment:
Eventually, we'd want in-toto-golang to use these protos, too, though, I think. From that standpoint, what would be most convenient?

Reply:
I can really go either way.  I actually think that the generated code _is not_ an example, and I hope people can use it directly.  We should see what @marcelamelara thinks because her intuition was that the go code should be separated.

Reply:
The way it's defined here, folks (including in-toto-golang) will be able to import the generated code.  They'll just need to use the correct import path.  The difference is really if they import `"github.com/in-toto/attestation/go/spec/v1.0-draft"` or `"github.com/in-toto/attestation/spec/v1.0-draft"`.

Reply:
My main reasoning behind separating the go code was because I'm imagining having pre-generated code for other languages as well in the future (Rust, Python etc.), and having them all live with the protos seemed potentially problematic. Could we simplify the import path to something like `github.com/in-toto/attestation/go` and `github.com/in-toto/attestation/go/predicates`, respectively, or does this go against golang conventions? (I'm not a big go coder).

Reply:
It might be possible and I can try once we figure out where we want the generated code to live.

For other languages I think best practices for them isn't to check the code in but just to publish to their respective distribution systems (e.g. pip).  Though I'm not sure about Rust.

Reply:
I should add that I don't think having 'go' or 'spec' in the import path is that bad, and I do think it's a nice property that the code is separated.  So yeah, doesn't really matter much to me.

Reply:
I propose that we keep this as is.  Any objections?

Reply:
Nope, what we have now is fine by me.